tests/main/ubuntu-core-classic/tasks.yaml

```diff
@@ -0,0 +1,34 @@
+summary: Ensure classic dimension works correctly
+systems: [ubuntu-core-16-64]
+execute: |
+    echo "Ensure classic can be installed"
+    snap install --devmode --beta classic
+    snap list|grep classic
+
+    echo "Check that classic can run commands inside classic"
+    classic test -f /var/lib/dpkg/status
+
+    echo "Ensure that after classic exits no processes are left behind"
+    classic "sleep 1337&"
+    if ps afx|grep 1337 ; then
+        echo "The sleep process was not killed when classic exited"
+        echo "Something is wrong with the cleanup"
+        exit 1
+    fi
+
+    echo "Ensure sudo works without a password inside classic"
+    if [ "$(classic sudo id -u)" != "0" ]; then
+        echo "sudo inside classic  did not work as expected"
+        exit 1
+    fi
+
+    for d in /proc /run /sys /dev /snappy; do
+        if ! classic -d $d; then
```

Comment:
Gar, silly me! Its `tests.yaml` currently but spread of course needs `task.yaml` (no extra `s`). Sorry for that! Fixing now.